docs/installation/frameworks/vuejs-v3.md

```diff
@@ -235,15 +235,59 @@ Finally, import the build in your application:
 
 ## Using CKEditor from source
 
-Integrating the rich text editor from source allows you to use the full power of the {@link framework/index CKEditor 5 Framework}.
+Integrating the rich text editor from source allows you to use the full power of the {@link framework/index CKEditor 5 Framework}. You have two options regarding building your application: Vite or Webpack.
+
+### Vite
+
+This guide assumes that you use `create-vue` as your boilerplate. To get started with Vite and Vue, run the command below.
+
+```bash
+npm init vue@latest ckeditor5-vue-example
+```
+
+This command will install and execute `create-vue`, the official project scaffolding tool for Vue. It will also allow you to customize your project, for example, by adding Typescript. Choose your preferred options.
+
+#### Installing necessary packages
+
+You need two packages to use CKEditor from source with Vue and Vite: the official Vue component and the Vite plugin.
+
+<info-box>
+	Using the Vite plugin to build CKEditor 5 from the source in Vite is still in the experimental phase. We encourage you to test it and give us feedback. To read more about integration with Vite or its limitations, check the {@link installation/advanced/integrating-from-source-vite Integrating from source with Vite} guide.
+</info-box>
+
+Install necessary packages alongside the default theme using the following command.
+
+```bash
+npm install --save @ckeditor/vite-plugin-ckeditor5 @ckeditor/ckeditor5-vue @ckeditor/ckeditor5-theme-lark
```

Comment:
Same case as in the Vue2 guide. Do we need to include `@ckeditor/ckeditor5-theme-lark` here? I know that this is used in `vite.config.js`, but the same is in `vue.config.js` section, where theme-lark is used, but installed in the next section. Removing theme-lark from here will make "Using the editor from source" section universal for both methods.